nmostesting/suites/IS1301Test.py

```diff
@@ -0,0 +1,316 @@
+# Copyright (C) 2023 Advanced Media Workflow Association
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+The script implements the IS-13 test suite as specified by the nmos-resource-labelling workgroup.
+At the end of the test, the initial state of the tested unit is supposed to be restored but this
+cannot be garanteed.

```

Comment:
Well done for commenting on the provenance of this test suite.  I'm guessing the spec if the true "source of truth" here.  Where possible it would be great to tie test cases back to normative language in the specifications (you can supply a `link=` to the test failure case that will embed a url link in the test results seen by the user).
```suggestion
The script implements the IS-13 test suite as specified by the AMWA IS-13 NMOS Annotation Specification (https://specs.amwa.tv/is-13/).
At the end of the test, the initial state of the tested unit is supposed to be restored but this
cannot be guaranteed.

```